src/main/resources/db/changelog/changes/001-create-books-table.yaml

```
@@ -0,0 +1,46 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-books-table
+      author: Anatolii Olkhovskyi
+      changes:
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: title
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: author
+                  type: varchar(100)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: isbn
+                  type: varchar(17)
```

```suggestion
                  type: char(17)
```
isbn has fixed length